test/game/games.test.js

```diff
@@ -0,0 +1,160 @@
+import { expect } from 'chai';
+import supertest from 'supertest';
+import app from '../../server';
+
+const request = supertest;
+let userToken;
+
+describe('Games API Endpoint', () => {
+  const user = {
+    email: `user${Math.random()}@gmail.com`,
+    username: 'my_user',
+    password: 'Mypassword1',
+    imageUr: 'https://www.mmm.png',
+  };
+
+  before((done) => {
+    request(app)
+      .post('/api/auth/signup')
+      .send(user)
+      .end((err, res) => {
+        userToken = res.body.token;
+        done();
+      });
+  });
+
+  describe('/api/games/:gameID/start', () => {
+    it('should not save a game if the user is not authenticated', (done) => {
+      request(app)
+        .post('/api/games/v5gty/start')
+        .set('card-game-token', '')
+        .send({
+          players: [{
+            username: 'Amarachi',
+            points: 4,
+          },
+          {
+            username: 'Amarachi',
+            points: 4,
+          }],
+          winner: 'Amarachi',
+          gameStarter: 'Luke',
+          roundsPlayed: 11,
+        })
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(403);
+          expect(res.body.message).to.deep.equal('You need to sign up or login');
+          done();
+        });
+    });
+
+    it('should save a game if the user is authenticated with a token', (done) => {
+      request(app)
+        .post('/api/games/v5gty/start')
+        .set('card-game-token', `${userToken}`)
```

Comment:
I don't think using backtick here is necessary, you can use the variable directly without it

Reply:
Great job so far implementing this feature. kudos